L1Trigger/L1THGCal/src/HGCalTriggerTools.cc

```diff
@@ -1,22 +1,87 @@
-
 #include "L1Trigger/L1THGCal/interface/HGCalTriggerTools.h"
+
+
 #include "L1Trigger/L1THGCal/interface/HGCalTriggerGeometryBase.h"
-#include "Geometry/Records/interface/CaloGeometryRecord.h"
+
 #include "DataFormats/ForwardDetId/interface/HGCalDetId.h"
+#include "DataFormats/HcalDetId/interface/HcalDetId.h"
+#include "Geometry/HGCalGeometry/interface/HGCalGeometry.h"
+#include "Geometry/HcalTowerAlgo/interface/HcalGeometry.h"
+#include "Geometry/CaloGeometry/interface/CaloGeometry.h"
+#include "Geometry/Records/interface/CaloGeometryRecord.h"
+
+#include "FWCore/Framework/interface/ESHandle.h"
+
+#include "FWCore/Framework/interface/EventSetup.h"
+
+
+namespace {
+  constexpr char hgcalee_sens[] = "HGCalEESensitive";
+  constexpr char hgcalfh_sens[] = "HGCalHESiliconSensitive";
+
+  constexpr std::float_t idx_to_thickness = std::float_t(100.0);
```

Comment:
this value doesn't seem to be used anywhere

Reply:
Removed